data/free worlds reconciliation.txt

```diff
@@ -1633,14 +1635,65 @@ mission "FW Pug 3C"
 		dialog "You have reached <planet>, but part of your fleet is not here yet. Better depart and wait for them to arrive in this star system."
 
 
+mission "FW Pug 3C(Extra ship)"
```

Comment:
There should be a space between the mission and the parenthesis. I'd also suggest that "extra ship" either be fully uppercase or lowercase, not a mix.
Or, we can change the mission name to something like "FW Pug 3.1C" or whatever. Same for the other missions.

Reply:
Oh, the "Extra ship" capitalisation was a typo. I'm going to try to change that, and try to figure out how to change the PR accordingly

Reply:
OK, done

Reply:
You still didn't add a space before the parenthesis for any of the missions.